lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -501,13 +501,26 @@
                     return this.canEditTeam();
                 },
 
-                /**
-                 * Returns whether _any_ of the topics are instructor-managed.
-                 */
-                anyInstructorManagedTopics: function() {
-                    return this.topicsCollection.some(
-                        function(topic) { return topic.isInstructorManaged(); }
-                    );
+                canViewBrowseTab: function() {
```

Comment:
This sounds like user doesn't have auth/permissions to view. Optional name change?
```suggestion
                shouldShowBrowseTab: function() {
```